### DISCUSSION TITLE
List outgoing/realised payments that have no externalId attached

### DISCUSSION THREAD

**Discussion by gzuuus** [ACINQ/phoenixd#23] [Q&A] [answered]
Is there a way to list outgoing/realised payments that have no externalId attached?

Thanks before hand!

**Comment by dpad85** [comment, marked as answer]
No there's no endpoint available to just list payments (outgoing or incoming).

We want to keep things as simple as possible, but maybe that's a bit too simple. For such a method, there needs to be some kind of filtering (maybe by date), with pagination and sensible default values. Do you want to give a shot at it and make a PR?

**Reply by gzuuus** [reply]
That would be awesome, otherwise you would have to develop a utility to store the payments in a separate db for those accounting reasons, which is also valid. 
I'm not a kotlin developer, I really want to learn the language, maybe I'll take it on a weekend I have free :)

**Comment by dpad85** [comment]
Fixed [with v0.1.5](https://github.com/ACINQ/phoenixd/releases/tag/v0.1.5), you can now list incoming and outgoing payments regardless of the external id (also with support for paging and date filtering).
